tools/lib-alert-tree/lib_alert_tree/kubernetes.py

```diff
@@ -3,14 +3,24 @@
 from . import models
 
 
+def container_alerts(name, severity="warning", namespace="default"):
+    """Common alerts for Containers."""
+    return [
+        models.ExistingAlert(
+            alertname, severity=severity, namespace=namespace, container=name
+        )
```

Comment:
You need to support pod name here as well as container name otherwise the call from `pod_alerts` will not give what you expected, (container name is not necessarily equal to the pod name)

Reply:
Done !!!